libs/platform/list/list.component.ts

```diff
@@ -513,10 +517,16 @@ export class ListComponent<T>
 
         this._selectionModel = new SelectionModel<BaseListItem>(this._multiSelect, this.selectedItems);
 
-        this._selectionModel.changed.pipe(takeUntilDestroyed(this._destroyed)).subscribe(() => {
+        this._selectionModel.changed.pipe(takeUntilDestroyed(this._destroyed)).subscribe((selection) => {
             this.selectedItems = this._selectionModel.selected;
             const event = new SelectionChangeEvent();
             event.selectedItems = this.selectedItems;
+            if (selection.added?.length) {
+                event.added = selection.added[0];
```

Comment:
So, this change does fix the issue we are seeing for the approval flow component. However, I think this code could pose a problem in the future for the list component, agnostic to the approval flow. It is interesting that `added` on a selectionChangeEvent is always just one `BaseListItem` but selectionModel emits an array of them.  Which makes me think, instead of doing `event.added = selection.added[0];` we need to either:
- Iterate over `selection.added` and emit a `selectedItemChange` event for each `selection.item`
- Or, update the `selectedItemChange` event to be able to pass either a single item (as it is now), or emit an array of them

same idea goes for `selection.removed` as well